projects/pytorch-py/Dockerfile

```diff
@@ -54,3 +54,4 @@ RUN /build_text.sh
 
 # Prepare initial corporas.
 RUN unzip load_fuzz.zip
+RUN unzip parse_native_yaml.zip
```

Comment:
We don't want to hold corpus in repo. Use download links to get corpus, or collect inputs from target repository.